service-worker/index.js

```diff
@@ -38,8 +39,16 @@ self.addEventListener('install', (event) => {
           return fetch(request)
             .then((response) => {
               if (response.status >= 400) {
-                throw new Error(`Request for ${url} failed with status ${response.statusText}`);
+                let error = new Error(`Request for ${url} failed with status ${response.statusText}`);
+
+                if (LENIENT_ERRORS) {
+                  console.warning(`Not caching ${url} due to ${error}`);
```

Comment:
Can we match this message with the other error message?

Reply:
It's the same error message, it uses the error above to fill in the error portion.